.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,53 @@
+name: Prepare Release
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
Does this need a condition?  Will this run for every pull request?

Reply:
Needed this to test the pipeline before the PR is merged. Will remove it before merging. 

`prepare-release` is only triggered by Release Ally app.

Reply:
Removed pull_request